app/assets/stylesheets/patterns/_ensure-word-wrap.scss

```diff
@@ -0,0 +1,17 @@
+.ensure-word-wrap {
+  /* These are technically the same, but use both */
+  overflow-wrap: break-word;
```

Comment:
Properties should be ordered -moz-hyphens, -ms-hyphens, -webkit-hyphens, hyphens, overflow-wrap, -ms-word-break, word-break, word-break, word-wrap